README.md

```diff
@@ -20,7 +20,7 @@ and renders declaratively using `lit-html`.
     if you're using a compiler that supports them, like [TypeScript](https://www.typescriptlang.org/) or [Babel](https://babeljs.io/).
```

Comment:
Suggest removing Babel reference until the included decorators support the TC39/Babel API.

Reply:
I believe that the current decorators are supported in Babel 7 with the `legacy `  flag. I'd link to the Babel docs: https://babeljs.io/docs/en/babel-plugin-proposal-decorators

Reply:
Done.